src/plugins/intel_npu/src/plugin/src/plugin.cpp

```diff
@@ -687,10 +687,10 @@ std::shared_ptr<ov::ICompiledModel> Plugin::compile_model(const std::shared_ptr<
     std::shared_ptr<ov::ICompiledModel> compiledModel;
     try {
         bool profiling = localConfig.get<PERF_COUNT>();
-        compiledModel = std::make_shared<CompiledModel>(model,
+        compiledModel = std::make_shared<CompiledModel>(std::move(model),
                                                         shared_from_this(),
-                                                        device,
-                                                        getCompiler(localConfig),
+                                                        std::move(device),
+                                                        std::move(getCompiler(localConfig)),
```

Comment:
No need to move from a function call - returned value, unless it's assigned to a variable is temporary, which is treated as rvalue already

Reply:
done, thx